update/database/postgresql/7.0/dbupdate-5.4.0-to-7.0.0.sql

```diff
@@ -0,0 +1,4 @@
+UPDATE ezsite_data SET value='7.0.0' WHERE name='ezpublish-version';
```

Comment:
Should this be 6.12?

Reply:
yes

Reply:
@emodric @andrerom You mean set to `6.12.0` in file `*/6.12/dbupdate-5.4.0-to-6.12.0.sql`, and same for mysql, right?

Reply:
Yep, exactly.